toolkit/templates/forms/_checkbox-tree-selection-buttons.html

```diff
@@ -34,14 +34,7 @@ <h3 class="categories-heading">{{ parent_category.label }}</h3>
                 last=loop.last,
                 hint=None
               %}
-
-                {# TODO: PLS FIX #}
-                {% if PAGES_BUILDER %}
-                  {% include 'forms/_selection-button.html' %}
-                {% else %}
-                  {% include 'toolkit/forms/_selection-button.html' %}
-                {% endif %}
-
+                {% include 'toolkit/forms/_selection-button.html' %}
               {% endwith %}
```

Comment:
My other pull request builds some internal pages that are useful for testing layout changes, so the include paths in there don't need to be usable in both contexts.  I put up the pull request to show what the new code is doing, but I think at this point it's probably better to close it until the macros refactoring is finished.